src/StaticWebAssetsSdk/benchmarks/Microsoft.NET.Sdk.StaticWebAssets.Benchmarks.csproj

```diff
@@ -0,0 +1,33 @@
+<Project>
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net10.0</TargetFramework>
+    <Description>Benchmarks for Microsoft.NET.Sdk.StaticWebAssets</Description>
+    <RootNamespace>Microsoft.NET.Sdk.StaticWebAssets.Benchmarks</RootNamespace>
+    <IsPackable>false</IsPackable>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+  </PropertyGroup>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
```

Comment:
Why do you import Sdk.props and Sdk.targets manually like this instead of using `<Project Sdk="..."/>`?